SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -598,9 +598,23 @@ void USpatialWorkerConnection::ProcessOutgoingMessages()
 		{
 			FCreateEntityRequest* Message = static_cast<FCreateEntityRequest*>(OutgoingMessage.Get());
 
+#if TRACE_LIB_ACTIVE
+			// We have to unpack these as Worker_ComponentData is not the same as FWorkerComponentData
+			TArray<Worker_ComponentData> UnpackedComponentData;
+			UnpackedComponentData.SetNum(Message->Components.Num());
+			for (int i = 0, Num = Message->Components.Num(); i < Num; i++)
```

Comment:
These are both arrays, can we just append? Or better yet, move the data?

Reply:
They are POD types so even moving would result in copy :(